rules/practice-makes-perfect/rule.md

```diff
@@ -27,29 +30,41 @@ These 5 steps can improve the delivery of a speech immensely (inspired by [Vinh
 
 <!--endintro-->
 
-1. Record the "test please".
-2. Do an audio audit – play the recording, only listen to the audio (don’t look at the video).
+1. Record the "Test Please".
+2. Do an audio audit – play the recording, only listen to the audio (don’t look at the video). 👀 Things to check: 
 
-   👀 Check: are you speaking too fast/too slow? Are you pausing appropriately? Voice stressing/pausing on important points? Are you too loud/too soft?
+   - Are you speaking too fast/too slow? 
+   - Are you pausing appropriately? 
+   - Voice stressing/pausing on important points? 
+   - Are you too loud/too soft?
 
-   ::: info
-   Tip: PowerPoint’s "Rehearse with Coach" will give you immediate feedback when practising.
-   :::
+::: info
+Tip: PowerPoint’s "Rehearse with Coach" will give you immediate feedback when practising.
+:::
 
-3. Do a visual audit – play the recording, this time looking at the video only (turn down the volume to zero, so that you can’t hear the audio).
+3. Do a visual audit – play the recording, this time looking at the video only (turn down the volume to zero, so that you can’t hear the audio). 👀 Things to check: 
```

Comment:
```suggestion
3. Do a visual audit – play the recording, this time looking at the video only (turn down the volume to zero, so that you can’t hear the audio).
   👀 Things to check: 
```